compiler/rustc_target/src/spec/wasm32_unknown_emscripten.rs

```diff
@@ -1,16 +1,12 @@
 use super::{cvs, wasm_base};
-use super::{LinkerFlavor, PanicStrategy, Target, TargetOptions};
+use super::{LinkArgs, LinkerFlavor, PanicStrategy, Target, TargetOptions};
 
 pub fn target() -> Target {
-    let mut options = wasm_base::options();
-
-    // Rust really needs a way for users to specify exports and imports in
-    // the source code. --export-dynamic isn't the right tool for this job,
-    // however it does have the side effect of automatically exporting a lot
-    // of symbols, which approximates what people want when compiling for
-    // wasm32-unknown-unknown expect, so use it for now.
-    options.add_pre_link_args(LinkerFlavor::Gcc, &["--export-dynamic"]);
```

Comment:
Could you explain this change, was it maybe that `--export-dynamic` was not needed for this target ?

Reply:
`LinkerFlavor::Gcc` is never used with emscripten, this code was a result of some mechanical copy-paste, see https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/emscripten.3A.20Is.20emcc.20still.20needed.3F/near/285406708